apps/scoutgame/app/robots.ts

```diff
@@ -1,10 +1,15 @@
+import { getPlatform } from '@packages/mixpanel/platform';
 import type { MetadataRoute } from 'next';
 
 export default function robots(): MetadataRoute.Robots {
+  const platform = getPlatform();
+  const isTelegram = platform === 'telegram';
```

Comment:
suggestion: call this variable "disallowRobots"

Reply:
yeah, better. Done